drafts/inlinemacros.md

```diff
@@ -144,6 +148,7 @@ positions:
 
         inline val x: inline T = E
 
+
```

Comment:
Why do we need to silently convert from something that the user has written, i.e. `: T`, to something that they haven't, i.e. `: inline T`? Won't the type inference take care of assigning the correct type here?